### PR TITLE
Update README badges to match psi4 style

### PR DESCRIPTION
**Pull request by jchodera**
This PR splits the anaconda cloud badges onto their own line, adding  badge that displays the latest conda package version available.

### CODE REVIEW

**Comment by jchodera** [comment]
image

**Comment by jchodera** [comment]
Hm, maybe we should adopt the psi4 style?
image

**Comment by davidlmobley** [comment]
I like the Psi4 style.

**Comment by jchodera** [comment]
Here's a [new live preview](https://github.com/openforcefield/openforcefield/blob/update-readme/README.md):
image
The "license: not identifiable by github" should be fixed with the update of the `LICENSE` file to a more standard format.

**Comment by jchodera** [comment]
It looks like the test that checks URLs are valid is mistakenly truncating URLs at underscores (`_`):
```
=================================== FAILURES ===================================
_____________ test_readme_links[https://img.shields.io/badge/dev] ______________
readme_link = 'https://img.shields.io/badge/dev'
    @pytest.mark.parametrize('readme_link', find_readme_links())
    def test_readme_links(readme_link):
        """Test the example"""
        from urllib.request import Request, urlopen
        # Some websites do not accept requests that don't specify the
        # client and the type of accepted documents so we add fake info
        # to avoid the response being an error.
        headers = {'User-Agent':'Mozilla/5.0',
                   'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8',}
        request = Request(readme_link, headers=headers)
>       urlopen(request)
openforcefield/tests/test_examples.py:134: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:223: in urlopen
    return opener.open(url, data, timeout)
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:532: in open
    response = meth(req, response)
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:642: in http_response
    'http', request, response, code, msg, hdrs)
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:570: in error
    return self._call_chain(*args)
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:504: in _call_chain
    result = func(*args)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <urllib.request.HTTPDefaultErrorHandler object at 0x10c9d4518>
req = <urllib.request.Request object at 0x11953fef0>
fp = <http.client.HTTPResponse object at 0x11953fda0>, code = 404
msg = 'Not Found', hdrs = <http.client.HTTPMessage object at 0x119531208>
    def http_error_default(self, req, fp, code, msg, hdrs):
>       raise HTTPError(req.full_url, code, msg, hdrs, fp)
E       urllib.error.HTTPError: HTTP Error 404: Not Found
../../../miniconda/envs/test/lib/python3.6/urllib/request.py:650: HTTPError
```
@andrrizzi : Any ideas?

**Comment by j-wags** [comment]
Pushed a potential fix. Worked when I ran tests locally. Let's see if Travis likes it.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-[![Funding](https://img.shields.io/badge/Funding-Open%20Force%20Field%20Consortium-brightgreen.svg)](http://openforcefield.org)
+| **Status** | [![Travis build](https://img.shields.io/travis/openforcefield/openforcefield/master.svg?logo=linux&logoColor=white)](https://travis-ci.org/openforcefield/openforcefield) [![Codecov coverage](https://img.shields.io/codecov/c/github/openforcefield/openforcefield.svg?logo=Codecov&logoColor=white)](https://codecov.io/gh/openforcefield/openforcefield) [![LGTM analysis](https://img.shields.io/lgtm/grade/python/g/openforcefield/openforcefield.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/openforcefield/openforcefield/context:python) |
+| :------ | :------- |
+| **Latest Release** | [![Last release tag](https://img.shields.io/github/release-pre/openforcefield/openforcefield.svg)](https://github.com/openforcefield/openforcefield/releases)  [![Commits since release](https://img.shields.io/github/commits-since/openforcefield/openforcefield/0.2.1.svg)](https://github.com/openforcefield/openforcefield/releases/tag/0.2.1) |
```
[the remark this answers is not in the document]
Comment (jchodera) [reply README.md]: Sounds good! Go for it!

File: LICENSE [hunk 2]
```diff
@@ -1,12 +1,4 @@
...
-More information on this license can be found at the Open Source Initiative
-
-https://opensource.org/licenses/MIT
-
-The complete license text is reproduced below:
-
----
+MIT License
 
 Copyright (c) 2016-2018, Open Force Field Initiative
```
[the remark this answers is not in the document]
Comment (jchodera) [reply LICENSE]: Sure!

**Comment by j-wags** [comment]
Hmm, so we actually have several Zenodo DOIs. 

[DOI](https://doi.org/10.5281/zenodo.597754) is the "concept" DOI. Clicking this will always resolve to the most recent release.

[DOI](https://doi.org/10.5281/zenodo.2638379) is the actual most recent release (0.2.1, note the different number).

So, basically we have this problem where the "auto-latest" zenodo badge will show a number that people shouldn't cite, because it loses version info. But, linking to a specific version number on the badge gives us a circular dependency during release, where we need to release to get the newest DOI, but then the README in the release points to the previous version's DOI. 

I guess the best thing to do is to keep the "concept" DOI there. People can't `Cmd-C` the number from the image, and clicking through will bring them to a page that indicates that versions have different DOIs.

**Comment by jchodera** [comment]
Good point. For the release DOIs, you can always go back to the release description pages and add those badges later by editing the Markdown text associated with the release:
https://github.com/openforcefield/openforcefield/releases

**Comment by j-wags** [comment]
Good idea. Let's add that to the release checklist.

Also, the license badge is still showing grey. I'll take a look at that...

**Comment by jchodera** [comment]
> Also, the license badge is still showing grey. I'll take a look at that...

It will be grey until the updated `LICENSE` file is merged into `master`.

**Comment by jchodera** [comment]
Also you will need to merge from `master` or rebase.

**Comment by j-wags** [comment]
Oh, got it. Merging.

**Pull request merged**